aspnetcore/fundamentals/servers/kestrel/sample2/KestrelDemo.csproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
we've been using */sample1/* and */sample2*
I think sample2 is clear enough. Additionally, the download/view sample links will point to the right version